src/main/java/cms/gov/madie/measure/resources/MeasureTransferController.java

```diff
@@ -196,4 +204,44 @@ private void setMeasureElmJsonAndErrors(Measure measure, String apiKey, String h
       measure.setCqlErrors(true);
     }
   }
+
+  protected void reorderGroupPopulations(List<Group> groups) {
```

Comment:
Please add a method javadoc explaining why the group populations need to be re-ordered. I will forget why we're doing this in about 3 days.

Reply:
@jkotanchik-SB added, thanks!